### DISCUSSION TITLE
Pricing and usage?

### DISCUSSION THREAD

**Discussion by pi43r** [pockethost/pockethost#201] [Q&A]
Hi!
I thought I would use your service for a quick PoC and art project that's going to run in June.
I couldn't find any usage limits on the website, but it's basically gonna be 1 user with occasional updates ...
Just wanted to make sure that I'm not getting surprised when we go live :goat:

**Comment by benallfree** [comment]
Hi @pi43r, the service is still in alpha and we aren't taking payments, so there are no service guarantees. Feel free to use PocketHost, but there are no promises :)

**Comment by pi43r** [comment]
Ok, thanks!
It was a breeze to set up :)
I might move it to another instance later, or gamble a little bit.
